File: backend/Application/Models/Dto/ListingDto/WarehouseDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using Domain.Enums;

namespace Application.Models.Dto.ListingDto
{
    public class WarehouseDto
    {
        public double CeilingHeight { get; set; }
        public bool LoadingDockAvailable { get; set; }
        public bool OfficeSpaceAvailable { get; set; }
        public List<WarehouseSuitableGoods> SuitableGoods { get; set; }
        public DateTime CreatedAt { get; set; }
        public DateTime UpdatedAt { get; set; }
        public Guid CommercialPropertyId { get; set; }
    }
}

File: mobile/lib/src/features/dashboard/domain/entities/warehouseDto.dart
class WarehouseDto {
  final int ceilingHeight;
  final bool loadingDockAvailable;
  final bool officeSpaceAvailable;
  final List<int> suitableGoods;

  WarehouseDto({
    required this.ceilingHeight,
    required this.loadingDockAvailable,
    required this.officeSpaceAvailable,
    required this.suitableGoods,
  });

  // from json
  
}
